### PR TITLE
Finished last functions for story three

### PR DESCRIPTION
**Pull request by kikiou1991**

### CODE REVIEW

File: src/components/HomePage/index.jsx [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+    setResults(true);
+  }
+
+  function toggleReset() {
+    setRecentJobs(true);
+    setResults(false);
+  }
+
+  console.log('results: ', results);
+  console.log('results recent: ', recentJobs);
```
Comment (ashleycoles) [inline src/components/HomePage/index.jsx]: Remove console.logs

File: src/components/RecentJobs/index.jsx [hunk 2]
```diff
@@ -5,54 +5,49 @@ import SkillsItems from '../SkillsItems';
...
+              <div className='flex justify-between px-3 text-white'>
+                <div className='text-left w-[45%]'>Job Title / Company</div>
+                <div className='flex justify-start w-[15%]'>Type</div>
+                <div className='flex justify-start w-[15%]'>Salary</div>
+                <div className='flex justify-start w-[25%]'>Skills</div>
+              </div>
+            </div>
+            <div>
+              {recentJobs.map((job) => (
+                <JobCard
```
Comment (ashleycoles) [inline src/components/RecentJobs/index.jsx]: I should hav epicked this up on the last story, but can you set each JobCard to have a cursor pointer on hover, just to indicate that they are interactive

File: src/components/SearchBar/index.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                <label className='text-white text-sm' htmlFor='contract'>
+                  Contract
+                </label>
+              </div>
+            </div>
+
+            <div>
+              <p className='text-white text-xs'>Popular skills:</p>
+            </div>
+          </div>
```
Comment (ashleycoles) [inline src/components/SearchBar/index.jsx]: Can you remove these for now please (feel free to save it somewhere), as these don't yet do anything, we don't want to have anything non-functional in main.

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Fantastic work team. Just a few comments for you

File: src/components/RecentJobs/index.jsx (hunk 2 above)
Comment (Ashton-Jeffrey) [reply src/components/RecentJobs/index.jsx]: Done

File: src/components/SearchBar/index.jsx (hunk 3 above)
Comment (Ashton-Jeffrey) [reply src/components/SearchBar/index.jsx]: Removed thanks

File: src/components/HomePage/index.jsx (hunk 1 above)
Comment (Ashton-Jeffrey) [reply src/components/HomePage/index.jsx]: Done

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**